components/comments/commentList.js

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import CommentItem from './commentItem';
+
+const CommentList = ({ value }) => {
+    return (
+        <>
+            <div className="row">
+                <div className="col">
+                    <h2>Comments</h2>
+                </div>
+                <div className="col text-right">
+                    <a
+                        href="#"
+                        className="button button-md button-primary button-leaf"
+                        id="addComment"
+                    >
+                        <span className="mdi mdi-plus"></span> Comment
+                    </a>
+                </div>
+            </div>
+            {value.map((comment) => {
+                return <CommentItem comment={comment} key={comment.identifier} />;
```

Comment:
```suggestion
                return <CommentItem {...comment} key={comment.identifier} />;
```

Is cleaner and also since is just a one-to-one with the content you passing, then in the component you can just destructure the object and get what you need.